src/models/tiles/tile-link-types.ts

```diff
@@ -1,6 +1,10 @@
+import { IDataSet } from "../data/data-set";
+
 export interface ITileLinkMetadata {
```

Comment:
I wasn't sure if it was the right move to expand this type or make another interface.  As far as I could see in practice, it represents metadata for a dataset.

Reply:
It's probably ok to keep this as is for now, but I agree that the meaning is getting diluted and it's probably worth considering better alternatives moving forward.